README.md

```diff
@@ -1 +1 @@
-# ImageStitching.jl
\ No newline at end of file
+# ImageStitching.jl
```

Comment:
A few badges can be helpful here, see also the ImageCore README https://github.com/JuliaImages/ImageCore.jl/blob/master/README.md